server/index-node/src/schema.graphql

```diff
@@ -14,6 +14,8 @@ type SubgraphIndexingStatus {
   failed: Boolean!
   error: String
   chains: [ChainIndexingStatus!]!
+  entityCount: BigInt!
```

Comment:
I'd like for this to be separate from the `SubgraphIndexingStatus` type.

Can we introduce a separate `SubgraphPricing` type for this with a `subgraphPricing(subgraphs: [String!]!): [SubgraphPricing!]!` resolver?